### PR TITLE
Inline Pmpermit + some fixes

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: userge/core/methods/decorators/raw_decorator.py [hunk 1]
```diff
@@ -90,9 +90,10 @@ def _clear_cht() -> None:
 async def _init(r_c: Union['_client.Userge', '_client._UsergeBot'],
                 r_m: RawMessage) -> None:
     global _U_ID, _B_ID  # pylint: disable=global-statement
-    if r_m.from_user and (r_m.from_user.is_self
-                          or (r_m.from_user.id in Config.SUDO_USERS)
-                          or (r_m.from_user.id in Config.OWNER_ID)):
+    if r_m.from_user and (
+        r_m.from_user.is_self or (
+            r_m.from_user.id in Config.SUDO_USERS) or (
+                r_m.from_user.id in Config.OWNER_ID)):
```
Comment (rking32) [inline userge/core/methods/decorators/raw_decorator.py:96]: it was clear before

File: userge/plugins/utils/pmpermit.py [hunk 2]
```diff
@@ -226,9 +229,17 @@ async def uninvitedPmHandler(message: Message):
                 f"You have {pmCounter[message.from_user.id]} out of 4 **Warnings**\n"
                 "Please wait until you get approved to pm !", del_in=5)
     else:
+        if userge.has_bot:
+            BOT = (await userge.bot.get_me()).username
```
Comment (rking32) [inline userge/plugins/utils/pmpermit.py:233]: can you use lower case for local variables. looks like a global variable. confusing ...

File: userge/plugins/utils/pmpermit.py [hunk 3]
```diff
@@ -245,3 +256,99 @@ async def outgoing_auto_approve(message: Message):
         {'_id': userID}, {"$set": {'status': 'allowed'}}, upsert=True)
     user_dict = await userge.get_user_dict(userID)
     await CHANNEL.log(f"**#AUTO_APPROVED**\n{user_dict['mention']}")
+
+
```
Comment (rking32) [inline userge/plugins/utils/pmpermit.py:260]: you should have to check that bot exists or not here. like `if userge.has_bot:` before calling `userge.bot.on_callback_query`

File: userge/core/methods/decorators/raw_decorator.py (hunk 1 above)
Comment (Krishna-Singhal) [reply userge/core/methods/decorators/raw_decorator.py:96]: but check pep8 not agree before, (line break before binary operator)

File: userge/plugins/utils/pmpermit.py (hunk 3 above)
Comment (Krishna-Singhal) [reply userge/plugins/utils/pmpermit.py]: yeah I forgot about that 👍

**Comment by rking32** [comment]
@Krishna-Singhal did you test this. not worked for me and had to fix so may errors

File: userge/plugins/utils/pmpermit.py [hunk 4]
```diff
@@ -286,15 +286,15 @@ async def pm_callback_block(_, c_q: CallbackQuery):
         owner = await userge.get_me()
         if c_q.from_user.id == owner.id:
             userID = int(c_q.matches[0].group(1))
-            user = await userge.get_users(userID)
+            await userge.block_user(userID)
```
Comment (Krishna-Singhal) [inline userge/plugins/utils/pmpermit.py:289]: Btw you blocking first, and sending blocked msg after blocking 😶😶

**Pull request merged**